metricbeat/mb/event.go

```diff
@@ -223,6 +223,9 @@ type PartialMetricsError struct {
 }
 
 func (p PartialMetricsError) Error() string {
+	if p.Err == nil {
+		return "Partial metrics error"
```

Comment:
```suggestion
		return ""
```

I think it is more correct to convert a nil error into the empty string, instead of making it look like it might have been initialized with a vague error.